active_learning_multiannotator/utils/active_learning.py

```diff
@@ -0,0 +1,92 @@
+"""
+This file defines the helper functions for the active learning simulation, which include data preperation
+for each round of active learning, and adding new annotators in the demonstration.
+"""
+import numpy as np
+import pandas as pd
+
+
+def setup_next_iter_data(
+    multiannotator_labels,
+    relabel_idx,
+    relabel_idx_unlabeled,
+    X,
+    X_unlabeled,
+    pred_probs,
+    pred_probs_unlabeled,
+    extra_labels=None,
+    extra_labels_unlabeled=None,
+):
+    """Updates inputs after additional labels have been collected in a single active learning round,
```

Comment:
throughout this notebook and code, let's try to refer to "active learning" as "multi-annotator active learning" or "active learning with multiple annotators", to more clearly distinguish our setup from the standard active learning one with a single label per example